testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/rx/rxjava2/Rx2FlowableProxyServerAsyncTest.java

```diff
@@ -0,0 +1,348 @@
+      List<byte[]> list = proxy.putBytes("3");
+      Assert.assertEquals(3, list.size());
+      for (byte[] b : list) {
+         Assert.assertTrue(Arrays.equals(Bytes.BYTES, b));
+      }
+   }
+
+   @Test
+   public void testPost() throws Exception {
+      List<String> list = proxy.post("a");
+      Assert.assertEquals(aStringList, list);
+   }
+
+   @Test
+   public void testPostThing() throws Exception {
+      List<Thing> list = proxy.postThing("a");
+      Assert.assertEquals(aThingList, list);
+   }
+
+   @Test
+   public void testPostThingList() throws Exception {
+      List<List<Thing>> list = proxy.postThingList("a");
+      Assert.assertEquals(aThingListList, list);
+   }
+
+   @Test
+   public void testPostBytes() throws Exception {
+      List<byte[]> list = proxy.postBytes("3");
+      Assert.assertEquals(3, list.size());
+      for (byte[] b : list) {
+         Assert.assertTrue(Arrays.equals(Bytes.BYTES, b));
+      }
+   }
+   
+   @Test
+   public void testDelete() throws Exception {
+      List<String> list = proxy.delete();
+      Assert.assertEquals(xStringList, list);
+   }
+
+   @Test
+   public void testDeleteThing() throws Exception {
+      List<Thing> list = proxy.deleteThing();
+      Assert.assertEquals(xThingList, list);
+   }
+   
+   @Test
+   public void testDeleteThingList() throws Exception {
+      List<List<Thing>> list = proxy.deleteThingList();
+      Assert.assertEquals(xThingListList, list);
+   }
+
+   @Test
+   public void testDeleteBytes() throws Exception {
+      List<byte[]> list = proxy.deleteBytes();
+      Assert.assertEquals(3, list.size());
+      for (byte[] b : list) {
+         Assert.assertTrue(Arrays.equals(Bytes.BYTES, b));
+      }
+   }
+   
+   @Test
+   @Ignore // @TODO Fix: see RESTEASY-1885.
+   public void testHead() throws Exception {
+      List<String> list = proxy.head();
+      Assert.assertNull(list);
+   }
+
+   @Test
+   public void testOptions() throws Exception {
+      List<String> list = proxy.options();
+      Assert.assertEquals(xStringList, list);
+   }
+
+   @Test
+   public void testOptionsThing() throws Exception {
+      List<Thing> list = proxy.optionsThing();
+      Assert.assertEquals(xThingList, list);
+   }
+
+   @Test
+   public void testOptionsThingList() throws Exception {
+      List<List<Thing>> list = proxy.optionsThingList();
+      Assert.assertEquals(xThingListList, list);
+   }
+
+   @Test
+   public void testOptionsBytes() throws Exception {
+      List<byte[]> list = proxy.optionsBytes();
+      Assert.assertEquals(3, list.size());
+      for (byte[] b : list) {
+         Assert.assertTrue(Arrays.equals(Bytes.BYTES, b));
+      }
+   }
+   
+   @Test
+   @Ignore // TRACE turned off by default in Wildfly
+   public void testTrace() throws Exception {
+      List<String> list = proxy.trace();
+      Assert.assertEquals(xStringList, list);
+   }
+
+   @Test
+   @Ignore // TRACE turned off by default in Wildfly
```

Comment:
TRACE can be enabled in Widlfly. Like in https://github.com/resteasy/Resteasy/blob/master/testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/TraceTest.java#L57.

Currently the test fails with TRACE enabled.

Reply:
@kanovotn, oh, cool, I didn't know about that. I'll look into it. Thanks.